pkg/proxy/proxy.go

```diff
@@ -81,7 +83,11 @@ type AppOverview struct {
 }
 
 func (p *Proxy) get(name, namespace string) (*release.Release, error) {
-	list, err := p.helmClient.ListReleases(helm.ReleaseListStatuses(releaseStatuses))
+	list, err := p.helmClient.ListReleases(
+		helm.ReleaseListLimit(p.listLimit),
```

Comment:
Do we want the limit here?

Is there any other way to retrieve the single release without relying on a paginated releases list? I am worried that the behavior in which we return not found but actually the page was too small

Reply:
I did a quick check in the helm/client.go and there does not seem to be a way, maybe we can use the client.ReleaseStatus but that one seems to only return release.info.

It seems that their CLI will not run into this issue because AFAICT they do the filtering (via the query parameter) after the list them all. So the pagination affects to the results, not to the sample in which they iterate.

Reply:
I missed that filter parameter. Using that it will just show the release we are interested in. Thanks!